realwork/WatchApp Extension/ExtensionDelegate.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ExtensionDelegate.swift
+//  WatchApp Extension
+//
+//  Created by Nathan Ngobale on 2020/04/29.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import WatchKit
+
+class ExtensionDelegate: NSObject, WKExtensionDelegate {
+
+    func applicationDidFinishLaunching() {
+        // Perform any final initialization of your application.
+    }
+
+    func applicationDidBecomeActive() {
+        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 194 characters](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5444460)